operator-common/src/main/java/io/strimzi/operator/PlatformFeaturesAvailability.java

```diff
@@ -103,7 +103,7 @@ private static OkHttpClient getOkHttpClient(KubernetesClient client)   {
 
         if (kubernetesVersion != null) {
             try {
-                futureVersion = Future.succeededFuture(new VersionInfo(Util.parseMap(kubernetesVersion)));
+                futureVersion = Future.succeededFuture(Util.parseVersionInfo(kubernetesVersion));
```

Comment:
What is the value of having this in Util and not right in this class? It does not seem to be used from other places.

Reply:
It was there before o.O

Reply:
Before you used a method from Util. But it was generic method used on many places. Now you added your own method (which is fine) used only in here and in the related test. So it does not mke sense anymore I guess?